crates/rspack_testing/src/test_config.rs

```diff
@@ -218,6 +222,8 @@ impl From<PxToRem> for PxToRemOptions {
 #[derive(Debug, JsonSchema, Deserialize)]
 #[serde(rename_all = "camelCase", deny_unknown_fields)]
 pub struct Output {
+  #[serde(default = "false_by_default")]
```

Comment:
If you want to use `false ` as the default value, you could just write `#[serde(default)]`.